src/makeRobot.js

```diff
@@ -38,7 +38,64 @@
  * @return {object}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  const Robot = {
```

Comment:
```suggestion
  const robot = {
```
для имен переменных лучше использовать Camel case.
можно newRobot.

Reply:
Igor is right. The first capital letter is used for React components. 
For the names of the variables star with lover-case letters